### ISSUE TITLE
Add ability to search for dashboards and alerts

### ISSUE THREAD

**Issue by jezdez** [mozilla/redash#274]
PR #270 added full-text search for queries but it may be useful to provide the same Postgres based search for the dashboards and alert views.

**Comment by washort** [comment]
Not sure this makes sense since the entire dashboard list is available client-side and there's no obvious search fields other than name.

**Comment by arikfr** [comment]
> the entire dashboard list is available client-side

I'm not sure it will be like this forever... 

>  there's no obvious search fields other than name.

There is an open pull request to add a description field & we can index widget titles and text boxes content.

**Comment by rafrombrc** [comment]
This all should be much better once the discovery branch lands upstream and then is merged into our repo.

**Comment by jezdez** [comment]
Moving this to m18 since this is less important than fixing the bugs.
